graph/common/Storage.java

```diff
@@ -27,11 +27,15 @@
 import com.vaticle.typedb.core.graph.iid.VertexIID;
 
 import javax.annotation.Nullable;
+import java.util.Arrays;
```

Comment:
unused import

Reply:
This has been fixed as per the last commit.